recipe/meta.yaml

```diff
@@ -33,7 +33,6 @@ about:
   home: http://ampl.com/
   license: BSD-3
   license_family: BSD
-  license_file: LICENSE
```

Comment:
This metadata information is mandatory for the package, please see the linter. The license file here **is** the one from the package.

Reply:
Thanks! I was not sure whether it should remain or not.